flink-runtime/src/test/java/org/apache/flink/runtime/util/JvmExitOnFatalErrorTest.java

```diff
@@ -137,9 +138,12 @@ public static void main(String[] args) throws Exception {
 
 			System.err.println("creating task");
 
+			TemporaryFolder temporaryFolder = new TemporaryFolder();
```

Comment:
Making it a `ClassRule` or a `Rule` will take care of the clean up.

Reply:
👍